deploy/README.md

```diff
@@ -8,79 +8,207 @@ npm run compile
 2. create `.env` file
 `cp .env.example .env`
 
-3. if it is necessary to have the bridge to relay native coins to an erc20 token, the following example could be used for `.env` file
+3. If it is necessary, deploy and configure a multi-sig wallet contract which will
+   be used to manage the bridge contracts after deployment.
+
+4. adjust parameters in `.env` file as per recommendation below depending on the desired bridge mode 
```

Comment:
adjust **the** parameters in **the** `.env` file ~~as per recommendation below~~ depending on the desired bridge mode <see recommendation below>